packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -14,11 +14,1064 @@ The Okta Administrator Roles API provides operations to manage administrative ro
 
 Explore the Administrator Roles API:  [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/4f1233beeef282acbcfb)
 
+## Custom role operations
+<ApiLifecycle access="beta" />
```

Comment:
We are not supposed to have Beta content in our publicly-available documentation.

Reply:
duplicated into a whole new doc - due to intersections between custom and standard roles we decided to have the new doc be a comprehensive doc combining both content and not just new Beta contents.